examples/level_zero_shared_memory/CMakeLists.txt

```diff
@@ -38,9 +38,7 @@ FetchContent_Declare(
     EXCLUDE_FROM_ALL)
 
 FetchContent_GetProperties(level-zero-loader)
```

Comment:
GetProperties is redundant, I believe: https://cmake.org/cmake/help/latest/module/FetchContent.html#command:fetchcontent_getproperties

Reply:
Missed these two, thx